learning_loop_node/detector/detector_node.py

```diff
@@ -236,15 +234,19 @@ async def _check_for_update(self) -> None:
                     self.detector_logic.load_model()
                     await self.send_status()
                     # self.reload(reason='new model installed')
-            else:
-                self.log.info('Versions are identic. Nothing to do.')
+
         except Exception as e:
             self.log.exception('check_for_update failed')
             msg = e.cause if isinstance(e, DownloadError) else str(e)
             self.status.set_error('update_model', f'Could not update model: {msg}')
             await self.send_status()
 
-    async def send_status(self) -> Union[str, Literal[False]]:
+    async def send_status(self) -> bool:
```

Comment:
I would rename this to something like `sync_status_with_learning_loop` since it is doing a lot more than sending its status now.
Similarly, I would make it raise an exception instead of returning a bool to make what is happening more clear.

Since it is public, that would be a breaking change though, so I supposed it is best to do this at some other point.

Reply:
I checked the yolo detector node and it is not accessed there (since it only implements `DetectorLogic`).
So I think we should change it now.
What do you think?